backend/README.md

```
@@ -34,3 +39,16 @@ Useful commands from the repository root:
 - `make backend-worker-run`
 
 The backend uses SQLAlchemy for ORM mappings and Alembic for schema migrations. `make backend-db-upgrade` applies the current schema to `BACKEND_DATABASE_URL`, which defaults to the local Postgres service defined in `.env.example`.
+
+## Auth routes and settings
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)